blocks/core/_lib/ff-tabs/_ff-tabs-core.js

```diff
@@ -113,16 +122,61 @@ function getSwapStatesMethod($root, options) {
                 addCompleteClasses($lastLinks, $lastContent);
             }
         }
+    }
+
+    function handleClick(e) {
+        e.preventDefault();
+        /*jshint validthis:true */
+        setState(this);
+    }
 
-    };
-}
 
-function setClickHandler($root, options) {
-    var linkSel = '[' + options.linkSelBase + ']';
-    $root.on('click', linkSel, getSwapStatesMethod($root, options));
+    function testBounds(value, length) {
+        if (value < 0) {
+            return length - 1;
+        } else if (value >= length) {
+            return 0;
+        } else return value;
+    }
+
+    function setActiveTab(index) {
```

Comment:
Ah, right. Not strictly no, but had gotten into the habit of exposing a fairly full API for testing (which haven't had a chance to do this time around, but would like to add them in at a later point, and since they're already there..?)